internal/handlers/tigris/tigrisdb/query.go

```diff
@@ -193,3 +201,41 @@ func BuildFilter(filter *types.Document) (string, error) {
 
 	return string(result), nil
 }
+
+// filterEqual returns the tigris filter with arguments that filters documents
+// where the value is equal to v.
+func filterEqual(v any) (filter any, err error) {
+	switch docVal := v.(type) {
+	case *types.Document, *types.Array, types.Binary,
+		types.NullType, types.Regex, types.Timestamp:
```

Comment:
I think this line is just impossible to reach, because we only call `filterEqual` on types `float64, string, types.ObjectID, bool, time.Time, int32, int64`